src/pages/index.js

```diff
@@ -84,9 +84,10 @@ function IndexPage() {
                 <h2>#VetsWhoCode</h2>
               </div>
               <p>
-                #VetsWhoCode is a veteran-led and operated 501(c)(3) charitable non-profit that
-                focuses on training veterans in programming and open source development with the
-                purpose of starting careers in the technology industry.
+                #VetsWhoCode is an inclusive, open-source, veteran-led and operated 501(c)(3) charitable public non-profit organization that
+                focuses on training veterans, active duty military, and military spouses in practical programming paradigms by using mentoring and open source development 
+                as the primary focus with the
+                goal of starting careers in the technology industry.
```

Comment:
VetsWhoCode is an inclusive, open-source, veteran-led and operated 501(c)(3) charitable public non-profit organization. We focus on training veterans, active duty military, and military spouses in practical programming paradigms by using mentoring and open source development with the goal of starting careers in the technology industry.